tests/TestCase/Utility/InflectorTest.php

```diff
@@ -204,80 +218,94 @@ public function testSingularizeMultiWordIrregular()
     /**
      * testInflectingPlurals method
      *
+     * @dataProvider pluralizeProvider
      * @return void
      */
-    public function testInflectingPlurals()
+    public function testInflectingPlurals($plural, $singular)
+    {
+        $this->assertEquals($plural, Inflector::pluralize($singular));
```

Comment:
My concern was about "what" input in case the expected output is the same for a few different inputs. But maybe that is not the case here.